XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityHand.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Interfaces.Providers.Controllers
+{
+    /// <summary>
+    /// Hand definition, used to provide access to hand joints and other data.
+    /// </summary>
+    public interface IMixedRealityHand : IMixedRealityController
```

Comment:
Missing IsTracked in the interface.

Reply:
Tracked should be part of the controller definition somewhere, we do have that.

Reply:
`IMixedRealityController` defines `IMixedRealityController.TrackingState` which can be used to check for `TrackingState == TrackingState.Tracked`